ChallengeTrackerAppAngular/src/app/program/create-new-program/create-new-program.component.ts

```diff
@@ -68,7 +69,7 @@ export class CreateNewProgramComponent implements OnInit {
 
 		//let startDateString: string = this.parseDate(this.startDate);
 		//let targetDateString: string = this.parseDate(this.targetCompletionDate);
-		
+		this.programManager = this.sessionService.getCurrentUser().userId;
 		if (createProgramForm.valid) 
```

Comment:
for readability, take out the code inside the if block and return early if createProgramForm is not valid

Reply:
done